src/cleanup_rules/swift/rules.toml

```diff
@@ -874,6 +874,150 @@ not_contains = [
     )""",
 ]
 
+#
+# Before 
+#   true == true
+#   (true) == true
+#   true == (true)
```

Comment:
can u write a rule to simplify a parenthesized expression to a non paramthesized expression ? 
The rule will become simpler then (look at Kotlin and Java rules). Basically one rule transforms `(true)` to `true`, then these equality / inequality rules do not have to account for parenthesis

Reply:
can u please add support for any expression for equaliity (look at the kotlin/java) rules . Equality rules do not have to be restricted to literals .